src/scanner.c

```diff
@@ -127,18 +127,18 @@ index_scanner_close(InternalScannerCtx *ctx)
  */
 static Scanner scanners[] = {
 	[ScannerTypeHeap] = {
-		.open = heap_scanner_open,
+		.opentable = heap_scanner_open,
```

Comment:
Sorry, I missed this before. Why add table ? It doesn't make a lot of sense as it is a common interface both for an index scan and a table scan.

Reply:
`open` is a macro on Windows. Happy to change to another name, but `open` is not usable.

Reply:
`openscanner`

Reply:
openscanner works for me.

Reply:
openheap or heapopen? (it is actually calling `heap_open()`)